index.html

```diff
@@ -8,7 +8,8 @@
     <meta name="author" content="Wolox">
     <link rel="stylesheet"  type="text/css" href="application.css">
   </head>
-  <body>
+  <body class="main-container">
```

Comment:
Use the container you add in texts-in-book-detail, you usaully don't  put classes to body